app/presenters/licence_details_presenter.rb

```diff
@@ -18,7 +18,20 @@ def local_authority_specific?
   def licence_authority_specific?
     !licence_details["location_specific"]
   end
-
+  
+  def any_action_uses_licensify?
+    authority["actions"] && authority["actions"].keys.uniq.any?{|chosen_action| uses_licensify(chosen_action) == true }
+  end
+  
+  def uses_licensify(chosen_action)
+    return false unless chosen_action
+    authority["actions"] && authority["actions"][chosen_action][0]['uses_licensify']
+  end
+  
+  def uses_licensify?
+    uses_licensify(action)
+  end
```

Comment:
I think it might be more consistent if we combine the "takes an argument" and "doesn't take an argument" versions here into:
```ruby
def user_licensify?(chosen_action = action)
  return false unless chosen_action
  authority["actions"] && authority["actions"][chosen_action][0]['uses_licensify']
end
```

I found it a little jarring down in the views where sometimes it was `uses_licensify(something)` and sometimes `uses_licensify?`.

Reply:
Combined